app/views/users/index.html.slim

```diff
@@ -0,0 +1,13 @@
+h1 Users
+
+table
+  thead
+    tr
+      th Username
+
+  tbody
+    - @users.each do |user|
+      tr
+        td = user.username
+  
+== link_to :Signup, signup_path
```

Comment:
How about using I18n localize here @gutakk 🤔 

https://guides.rubyonrails.org/i18n.html

Reply:
@andyduong1920
Could you please explain more why do we need l18n localize?

Reply:
Well basically we don't need in this project but you can look on that so you know how to use i18n localization in Rails.

For the real project, we always use I18n at the beginning, even that project only support 1 language at beginning because that will easy to adding another support language.

Let say the project starting with English, later on the client want to have Thai translation, instead for recheck all source code, we just adding a new `YML` file which contain Thai text :) 

So it's up to you, you can take a look on I18n localize in Rails now or later, I just want to introduce you what we are using in the real project :)

Reply:
Thanks for your concern, I will take a look at it and try to add to this PR

Reply:
Updated view to support i18n